ethcore/node_filter/src/lib.rs

```diff
@@ -20,9 +20,14 @@ extern crate ethcore;
 extern crate ethcore_bytes as bytes;
 extern crate ethcore_network as network;
 extern crate ethereum_types;
-extern crate native_contracts;
-extern crate futures;
 extern crate parking_lot;
+extern crate lru_cache;
+
+extern crate ethabi;
+#[macro_use]
+extern crate ethabi_derive;
+#[macro_use]
+extern crate ethabi_contract;
 
```

Comment:
Redundant.

Reply:
it's not. it exposes `use_contract!` macro which is used

Reply:
(I mean the new line :))

Reply:
kk :)